homeassistant/components/reolink/host.py

```diff
@@ -432,6 +432,15 @@ def unregister_webhook(self):
         webhook.async_unregister(self._hass, self.webhook_id)
         self.webhook_id = None
 
+    @property
+    def event_connection(self) -> str:
+        """Return the event connection type."""
+        if self._webhook_reachable:
+            return "onvif_push"
+        if self._long_poll_received:
+            return "onvif_long_poll"
+        return "fast_poll"
```

Comment:
This should probably not be a sensor but just part of diagnostics.

We don't allow calculating state in entities. These values don't come directly from the api but are just how Home Assistant interfaces with the api.

Reply:
opened https://github.com/home-assistant/core/pull/96903 to remove it.